components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -1712,13 +1712,13 @@ this mask will have smb calculated over the entire global land surface
 <!-- =========================================  -->
 <!-- Defaults for FATES interface -->
 <!-- =========================================  -->
-<use_fates_spitfire            use_ed=".true.">.false.</use_fates_spitfire>
-<use_fates_planthydro          use_ed=".true.">.false.</use_fates_planthydro>
-<use_fates_ed_st3              use_ed=".true.">.false.</use_fates_ed_st3>
-<use_fates_ed_prescribed_phys  use_ed=".true.">.false.</use_fates_ed_prescribed_phys>
-<use_fates_logging             use_ed=".true.">.false.</use_fates_logging>
-<use_fates_inventory_init      use_ed=".true.">.false.</use_fates_inventory_init>
-<fates_inventory_ctrl_filename use_ed=".true."> " " </fates_inventory_ctrl_filename>
+<use_fates_spitfire            use_fates=".true.">.false.</use_fates_spitfire>
+<use_fates_planthydro          use_fates=".true.">.false.</use_fates_planthydro>
+<use_fates_ed_st3              use_fates=".true.">.false.</use_fates_ed_st3>
+<use_fates_ed_prescribed_phys  use_fates=".true.">.false.</use_fates_ed_prescribed_phys>
+<use_fates_logging             use_fates=".true.">.false.</use_fates_logging>
+<use_fates_inventory_init      use_fates=".true.">.false.</use_fates_inventory_init>
+<fates_inventory_ctrl_filename use_fates=".true."> " " </fates_inventory_ctrl_filename>
 <use_vertsoilc       bgc_mode="ed"  >.true.</use_vertsoilc>
```

Comment:
should we change this to "fates"?

Reply:
ie the bgc_mode?

Reply:
Good catch. The namelist file and the build script are now updated.